CONTRIBUTING.md

```diff
@@ -4,7 +4,7 @@
 
 The GraphQL Accounts project was intended - since it's inception - to be a community maintained project.  We'd love to see you get involved (especially long time contributors from the Meteor community who we've worked with before).
```

Comment:
Also, the `devel` branch isn't being used to keep the latest development code so references to that should be removed.